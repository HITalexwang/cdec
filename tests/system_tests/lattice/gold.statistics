-lm_nodes 6
-lm_edges 10
-lm_paths 5
-lm_trans ab
-lm_trans -0.225
+lm_nodes 6
+lm_edges 10
+lm_paths 5
+lm_trans ab
