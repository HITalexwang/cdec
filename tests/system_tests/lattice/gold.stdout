0 ||| ab ||| WordPenalty=-0.434294 SourceWordPenalty=-0.434294 Cost=0.1 LatticeCost=0.125 ||| -1.09359
0 ||| cb ||| WordPenalty=-0.434294 SourceWordPenalty=-0.868589 Cost=0.3 LatticeCost=2.25 ||| -3.85288
0 ||| a_b ||| WordPenalty=-0.434294 SourceWordPenalty=-0.868589 Cost=0.2 LatticeCost=2.5 ||| -4.00288
0 ||| a' b ||| WordPenalty=-0.868589 SourceWordPenalty=-0.868589 Glue=1 Cost=0.3 LatticeCost=2.5 ||| -4.53718
0 ||| a b ||| WordPenalty=-0.868589 SourceWordPenalty=-0.868589 Glue=1 Cost=0.3 LatticeCost=2.5 ||| -4.53718
